src/lib/utils/apy.ts

```diff
@@ -2,8 +2,8 @@ import { IndexData } from '@/lib/utils/api/index-data-provider'
 
 export function calculateApy({
   APY = 0,
-  Rate = 0,
-  StreamingFee = 0,
+  ApyCost = 0,
+  ApyStreamingFee = 0,
 }: Partial<IndexData>) {
-  return (APY + Rate + StreamingFee) / 100
+  return (APY + ApyCost + ApyStreamingFee) / 100
```

Comment:
While we're add it, I'd advocate for moving this to the API using a convenience property. Or do we ever use those props by themselves somewhere?